bitnami/chainloop/templates/cas/deployment.yaml

```diff
@@ -0,0 +1,175 @@
+      labels: {{ include "chainloop.cas.labels" . | nindent 8 }}
+    spec:
+      {{- include "common.images.renderPullSecrets" (dict "images" (list .Values.cas.image) "context" $) | nindent 6 }}
+      serviceAccountName: {{ include "chainloop.cas.serviceAccountName" . }}
+      automountServiceAccountToken: {{ .Values.cas.automountServiceAccountToken }}
+      {{- if .Values.cas.hostAliases }}
+      hostAliases: {{- include "common.tplvalues.render" (dict "value" .Values.cas.hostAliases "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.cas.affinity }}
+      affinity: {{- include "common.tplvalues.render" ( dict "value" .Values.cas.affinity "context" $) | nindent 8 }}
+      {{- else }}
+      affinity:
+        {{- $podLabels := include "common.tplvalues.merge" (dict "values" (list .Values.cas.podLabels .Values.commonLabels) "context" .) }}
+        podAffinity: {{- include "common.affinities.pods" (dict "type" .Values.cas.podAffinityPreset "component" "cas" "customLabels" $podLabels "context" $) | nindent 10 }}
+        podAntiAffinity: {{- include "common.affinities.pods" (dict "type" .Values.cas.podAntiAffinityPreset "component" "cas" "customLabels" $podLabels "context" $) | nindent 10 }}
+        nodeAffinity: {{- include "common.affinities.nodes" (dict "type" .Values.cas.nodeAffinityPreset.type "key" .Values.cas.nodeAffinityPreset.key "values" .Values.cas.nodeAffinityPreset.values) | nindent 10 }}
+      {{- end }}
+      {{- if .Values.cas.nodeSelector }}
+      nodeSelector: {{- include "common.tplvalues.render" ( dict "value" .Values.cas.nodeSelector "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.cas.tolerations }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.cas.tolerations "context" .) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.cas.priorityClassName }}
+      priorityClassName: {{ .Values.cas.priorityClassName | quote }}
+      {{- end }}
+      {{- if .Values.cas.schedulerName }}
+      schedulerName: {{ .Values.cas.schedulerName | quote }}
+      {{- end }}
+      {{- if .Values.cas.topologySpreadConstraints }}
+      topologySpreadConstraints: {{- include "common.tplvalues.render" (dict "value" .Values.cas.topologySpreadConstraints "context" .) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.cas.podSecurityContext.enabled }}
+      securityContext: {{- omit .Values.cas.podSecurityContext "enabled" | toYaml | nindent 8 }}
+      {{- end }}
+      {{- if .Values.cas.terminationGracePeriodSeconds }}
+      terminationGracePeriodSeconds: {{ .Values.cas.terminationGracePeriodSeconds }}
+      {{- end }}
+      initContainers:
+        {{- if .Values.cas.initContainers }}
+          {{- include "common.tplvalues.render" (dict "value" .Values.cas.initContainers "context" $) | nindent 8 }}
+        {{- end }}
+      containers:
+        - name: {{ .Chart.Name }}
```

Comment:
Same comment here, we usually hardcode a container name, in this case we could use 'cas'.